### DISCUSSION TITLE
Facing Challenge During Installation

### DISCUSSION THREAD

**Discussion by avi-arora** [mcw0/DahuaConsole#27] [Q&A]
Hi, 

While installing, i'm facing challenge in unicorn package. 
can you please help me out how to setup this. i'll be forever grateful. 

Error screenshot is attached below. 

<img width="938" alt="Screenshot 2022-03-20 at 3 49 25 AM" src="https://user-images.githubusercontent.com/10464065/159140422-ed399be2-4ac9-45ae-8803-982dc89c14c7.png">

Kind Regards, 
Avi

**Comment by redowan99** [comment]
install wheel

**Reply by avi-arora** [reply]
I did, still not working @redowan99
